docs/UserGuide.md

```diff
@@ -52,23 +53,23 @@ your family tree and history.
 
    * `list` : Lists all contacts.
 
-   * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+   * `add /Name John Doe /Phone 98765432 /Email johnd@example.com /Address John street, block 123, #01-01` : Adds a contact with the specified attributes to the Address Book.
 
-   * `delete 3` : Deletes the 3rd contact shown in the current list.
+   * `delete /0001` : Deletes the contact with the `UUID` "0001".
```

Comment:
Future TODO: Introduce reader to Person UUID system, as a 4 character label that is unique to a person.